src/if_statements.py

```diff
@@ -353,55 +288,208 @@ def immediate_postdominator(context: Context, start: Node, end: Node) -> Node:
     return postdominators[0]
 
 
-def count_non_postdominated_parents(
-    context: Context, child: Node, curr_end: Node
-) -> int:
+def is_and_statement(next_node: Node, bottom: Node) -> bool:
+    return (
+        not isinstance(next_node, ConditionalNode)
+        or next_node not in bottom.parents
```

Comment:
I think `(next_node.conditional_edge is bottom or next_node.fallthrough_edge is bottom)` is clearer